cmd/pics/state.go

```diff
@@ -109,11 +108,74 @@ func keyTape(t *trie.Trie, number int) error {
 	return nil
 }
 
+func stateDatabaseMap(db* bolt.DB, number int) error {
+	filename := fmt.Sprintf("state_%d.dot", number)
+	f, err := os.Create(filename)
+	if err != nil {
+		return err
+	}
+	i := 0
+	visual.StartGraph(f, true)
+
+	if err := db.View(func(readTx *bolt.Tx) error {
+		return readTx.ForEach(func(name []byte, b *bolt.Bucket) error {
+			
+			if i == 0 {
```

Comment:
You don't even need this check here, either. Just the call `StartCluster`. Then, you can change `return b.ForEach(...)` on the line 130 to `if err := b.ForEach(..); err != nil { return err }`, and then just put `EndCluster` after that